modules/claims_api/spec/sidekiq/evidence_waiver_builder_job_spec.rb

```diff
@@ -1,21 +1,27 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require 'bgs_service/benefit_claim_service'
+require 'common/file_helpers'
 
-RSpec.describe ClaimsApi::EvidenceWaiverBuilderJob, type: :job do
+RSpec.describe ClaimsApi::EvidenceWaiverBuilderJob, type: :job, use_cassette: 'claims_api/bd/upload' do
   subject { described_class }
 
   before do
     Sidekiq::Job.clear_all
   end
 
   let(:ews) { create(:claims_api_evidence_waiver_submission, :with_full_headers_tamara) }
+  let(:bgs_claim) { create(:bgs_response) }
+  let(:output_path) { '' }
 
   describe 'generating the filled and signed pdf' do
     it 'generates the pdf to match example' do
```

Comment:
Is this text still correct (or was it ever?).  Seems like this test is for the upload and then file deletion.  Let me know if I am missing something though. 

I believe, if this text is correct, this scenario would also already be covered [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/spec/lib/claims_api/evidence_waiver_pdf_constructor/pdf_spec.rb)